content/help/ops/traffic-management/proxy-cmd/index.md

```diff
@@ -278,6 +278,16 @@ one route that matches on everything. This route tells Envoy to send the request
         }
     ]
     {{< /text >}}
+    
+1. To see the endpoints currently available for the cluster use the proxy-config endpoints command.
```

Comment:
Self-nit: To see the endpoints currently available for *this* cluster use the proxy-config endpoints command.